terraform/backend.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  backend "s3" {
+    region = "us-east-1"
+    key    = "426857564226/papila.tfstate"
```

Comment:
try not to reveal the account id. Do not push the backend file here. Use backend_override to check on local machine.